code/src/components/TodoRemoveall.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { useDispatch } from 'react-redux'
+import { todos } from "../reducers/todos.js"
+import { Button } from '../lib/Button'
+
+
+export const TodoRemoveall = () => {
+    const dispatch = useDispatch()
+    return (
+      <Button
+        onClick={() => {
+          dispatch(todos.actions.removeAll())
+        }}
+      >
+        Remove All
+      </Button>
+    )
+  }
```

Comment:
Good that it has it's own component. 

Maybe consider renaming to `TodoRemoveAll`
Since the naming is so specific it's okay that a dispatch happens here. 

If this was to be a reusable component, we would have to take the dispatch out of this component and rename it something more generic.